test/typings/tsconfig.5.2.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "./tsconfig.4.7.json"
```

Comment:
Does it make sense to extend `tsconfig.5.0.json`, if we have one? I feel 5.2 and 4.7 are rather far apart.